requirements.txt

```diff
@@ -6,6 +6,7 @@ colander==1.4
 colorama==0.3.9
 cornice==3.0.0
 cornice-swagger==0.5.2
+dockerflow==2017.5.0
```

Comment:
this won't work I think, since we need https://github.com/mozilla-services/python-dockerflow/pull/21